src/chainstate/stacks/boot/pox-2.clar

```diff
@@ -831,7 +892,15 @@
                     add-amount: increase-by })))
             (err ERR_STACKING_UNREACHABLE))
       ;; NOTE: stacking-state map is unchanged: it no longer tracks amount-stacked in PoX-2
-      (ok { stacker: tx-sender, total-locked: (+ amount-stacked increase-by)})))
+
+      (ok (merge event-info {
+            name: "stack-increase", 
```

Comment:
Can`pox-addr` be added to this payload? It appears to be the only function missing this other than `handle-unlock`

Reply:
Follow up question, should `pox-addr` be added to the "base" event data (i.e. `get-event-info`)?

Reply:
I think no for the latter question, since I get the value for `pox-addr` as a function parameter (it isn't always just a property that I can query with the stacker address)